app/cafe.py

```diff
@@ -0,0 +1,18 @@
+import datetime
+from app.errors import NotVaccinatedError, \
+    OutdatedVaccineError, NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError

```

Comment:
Could you add messages?)

Reply:
i added messages in errors.py in __init__